### PR TITLE
ECOSYS-99 Add user credential to start secured Payara Server from VSCode

### PR DESCRIPTION
**Pull request by jGauravGupta**
# Description
This is a feature providing interface from VSCode to update the admin credential and start a secured domain.

# Test

- Install Node and Yarn
- Open a terminal and run `yarn install`  to install all dependencies for the project.
- Run `code .` command from the terminal to open the project in VS Code.
- Press `F5` from Code to run the project which will start another instance of Code with an installed plugin.
- Press `Ctrl + Shift + P` and search for `Add Payara Server` option.
- Right-click on the server to update the credential for the secured domain:
image

### CODE REVIEW

**Comment by MattGill98** [comment]
Conflicts need resolving before I can review this

File: src/main/fish/payara/server/PayaraInstanceController.ts [hunk 1]
```diff
@@ -26,13 +26,14 @@ import * as fs from "fs";
 import * as tmp from "tmp";
 import * as fse from "fs-extra";
 import * as cp from 'child_process';
+import * as xml2js from 'xml2js';
```
Comment (Pandrex247) [inline src/main/fish/payara/server/PayaraInstanceController.ts]: Duplicate import, see line 24.

Comment (MattGill98) [reply src/main/fish/payara/server/PayaraInstanceController.ts]: It looks like a few merge commits went a bit funky, this needs sorting before a review

**Comment by jGauravGupta** [comment]
Sorry messed up in one of the master branch merge from Github UI.

File: package.json [hunk 2]
```diff
@@ -90,6 +91,11 @@
 				"category": "Payara"
 			},
 			{
+				"command": "payara.server.credential",
+				"title": "Update Credential",
```
[the remark this answers is not in the document]
Comment (jGauravGupta) [reply package.json]: Updated the naming of command.

**Pull request merged**